frontend/src/screens/home/home.tsx

```diff
@@ -13,10 +15,104 @@ interface HomeScreenProps {
 
 const StyledView = styled(View);
 const StyledText = styled(Text);
+const StyledImage = styled(Image);
+
+const WelcomeBlock = ({ name }: { name: string }) => {
```

Comment:
Maybe this was just here for testing purposes, but can we extract this WelcomeBlock to another file?